tests/nightly/test_np_large_array.py

```diff
@@ -568,6 +572,682 @@ def test_slice_assign():
+
+@use_np
+@pytest.mark.skip(reason='segfaults on large tensor ~2**30')
+def test_insert():
+    A = np.zeros((INT_OVERFLOW, 2))
+    B = np.ones((INT_OVERFLOW))
+    B[-1] = 2
+    C = np.insert(A, 1, B, axis=1)
+    assert C.shape == (INT_OVERFLOW, 3)
+    assert C[0, 1] == 1 and C[-1, 1] == 2
+
+@use_np
+@pytest.mark.skip(reason='segfaults on large tensor ~2**30')
+# problem might be on total size
+def test_interp():
+    xp = np.array([1, 2, 3])
+    fp = np.array([3, 2, 1])
+    A = np.ones((2, INT_OVERFLOW))
+    A[-1, -1] = 2.5
+    A.attach_grad()
+    with mx.autograd.record():
+        B = np.interp(A, xp, fp)
+        B.backward()
+    assert B.shape == A.shape
+    assert B[-1, -1] == 1.5
+    assert A.grad.shape == A.shape
+    assert A.grad[-1, -1] == 0
+
+@use_np
+def test_invert():
+    A = np.zeros((2, INT_OVERFLOW), dtype='uint8')
+    A[-1, -1] = 1
+    A.attach_grad()
+    with mx.autograd.record():
+        B = np.invert(A)
+        B.backward()
+    assert B.shape == A.shape
+    assert B[0, 0] == 255 and B[-1, -1] == 254
+    assert A.grad.shape == A.shape
+    assert A.grad[-1, -1] == 0
+
+@use_np
+def test_exp():
+    A = np.ones((2, INT_OVERFLOW))
+    A[-1, -1] = 2
+    A.attach_grad()
+    with mx.autograd.record():
+        B = np.exp(A)
+        B.backward()
+    assert B.shape == A.shape
+    assert_almost_equal(B[0, 0], np.array(np.e**1), rtol=1e-5, atol=1e-5)
+    assert_almost_equal(B[-1, -1], np.array(np.e**2), rtol=1e-5, atol=1e-5)
+    assert A.grad.shape == A.shape
+    assert_almost_equal(A.grad[-1, -1], B[-1, -1], rtol=1e-5, atol=1e-5)
+
+@use_np
+def test_expm1():
+    A = np.ones((2, INT_OVERFLOW))
+    A[-1, -1] = 2
+    A.attach_grad()
+    with mx.autograd.record():
+        B = np.expm1(A)
+        B.backward()
+    assert B.shape == A.shape
+    assert_almost_equal(B[0, 0], np.array(np.e**1 - 1), rtol=1e-5, atol=1e-5)
+    assert_almost_equal(B[-1, -1], np.array(np.e**2 - 1), rtol=1e-5, atol=1e-5)
+    assert A.grad.shape == A.shape
+    assert_almost_equal(A.grad[-1, -1], np.array(np.e**2), rtol=1e-5, atol=1e-5)
+
+@use_np
+@pytest.mark.skip(reason='segfaults on large tensor ~2**30')
+def test_power():
+    INT_OVERFLOW = 2**30
+    A = np.full((2, INT_OVERFLOW), 2)
+    B = np.ones((2, INT_OVERFLOW))
+    B[-1, -1] = 3
+    A.attach_grad()
+    B.attach_grad()
+    with mx.autograd.record():
+        C = np.power(A, B)
+        C.backward()
+    assert C.shape == A.shape
+    assert C[-1, -1] == 8
+    assert A.grad.shape == A.shape
+    assert A.grad[-1, -1] == 12
+    assert B.grad.shape == B.shape
+    assert_almost_equal(B.grad[-1, -1], 2**3 * np.log(2), rtol=1e-5, atol=1e-5)
+
+@use_np
+def test_frexp():
+    A = np.ones((2, INT_OVERFLOW))
+    A[-1, -1] = 9
+    B, C = np.frexp(A)
+    assert_almost_equal(A[-1, -1], B[-1, -1] * 2 ** C[-1, -1], \
+        rtol=1e-5, atol=1e-5)
+
+@use_np
+@pytest.mark.skip(reason='segfaults on large tensor ~2**30, likely the same \
```

Comment:
in general we prefer not to add code that's skipped at the time of adding. for this case, let's at least create an issue to track this

Reply:
I am going to fix those ops and enable the tests then. I will probably do this next week after I cover all the 200 something numpy ops that are exposed in the symbol api. We have an internal doc to track the progress but yeah I will create a public issue to list all failing ops as well

Reply:
shall we exclude these skipped tests for now then?

Reply:
Sure I will take them out. Will add them back along with fixes